pytorch/README.md

```diff
@@ -45,9 +45,10 @@ The images below additionally include [Jupyter Notebook](https://jupyter.org/) s
 
 | Tag(s)                | Pytorch  | IPEX          | Driver | Jupyter Port | Dockerfile      |
 | --------------------- | -------- | ------------- | ------ | ------------ | --------------- |
-| `xpu-jupyter`         | [v2.1.0] | [v2.1.30+xpu] | [803]  | `8888`       | [v0.4.0-Beta]   |
```

Comment:
What is this tag https://hub.docker.com/layers/intel/intel-extension-for-pytorch/xpu-jupyter/images/sha256-fcf8e51efb2d0e62e01a90052eee8b45b72882be907953a82510b59b82b77cc6?context=explore?

Reply:
that is a very old version of the image from 9 months ago and certainly not the latest.